drivers/mmcsd/mmcsd_sdio.c

```diff
@@ -1647,14 +1692,19 @@ static ssize_t mmcsd_readmultiple(FAR struct mmcsd_state_s *priv,
       return ret;
     }
 
-  /* Send STOP_TRANSMISSION */
+#ifndef CONFIG_MMCSD_MMCSUPPORT
```

Comment:
Remove ifdef because priv->cmd23support is true for MMC.

Reply:
The reason is as stated above